### ISSUE TITLE
Gutenberg 4.1 custom HTML gets unwanted <br>

### ISSUE THREAD

**Issue by manake** [WordPress/gutenberg#11031]
After v4.1 update my custom HTML blocks that were working suddenly got unwanted `<br>` tags even though there are no `<br>` tags in them.

Example block content:
```
<div class="x">
	<a href="/y/">y</a>
	<a href="/z/">z</a>
</div>
```

**Proposed solution:**
Remove automatic `<br>`.

**Comment by mcsf** [comment]
Hello. This needs concrete steps to follow so anyone can reproduce the issue.

**Comment by youknowriad** [comment]
Should be fixed by #11050

**Comment by manake** [comment]
Version 4.1.1 solved this for me.
